task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -41,40 +43,76 @@ spec:
       source /utils.sh
       trap 'handle_error $(results.TEST_OUTPUT.path)' EXIT
 
+      images_processed_template='{"image": {"pullspec": "'"$IMAGE_URL"'", "digests": [%s]}}'
       mkdir /manifests/ && cd /manifests/
 
-      image_with_digest="${IMAGE_URL}@${IMAGE_DIGEST}"
+      imagewithouttag=$(echo $IMAGE_URL | sed "s/\(.*\):.*/\1/" | tr -d '\n')
```

Comment:
nit: would `echo -n` be simpler here to avoid the newline?

Reply:
yes, absolutely!